app/models/post.rb

```diff
@@ -10,10 +10,15 @@ class Post < ApplicationRecord
 
   after_create do
     ActionCable.server.broadcast "posts_realtime", { row: PostsController.render(locals: {post: Post.last}, partial: 'post').html_safe }
+    ActionCable.server.broadcast "topbar", { review: Post.feedbacks_count }
   end
 
   after_create :autoflag
 
+  def self.feedbacks_count
+    Post.includes(:feedbacks).where( :feedbacks => { :post_id => nil }).count
```

Comment:
In theory, yeah, @j-f1. That does (unsurprisingly) a LEFT JOIN SQL query, which has all post records whether they have feedback or not - so you can filter down to those with none.